packages/js/api-core-tests/tests/shipping/shipping-zones.test.js

```diff
@@ -0,0 +1,97 @@
+const { shippingZonesApi } = require( '../../endpoints' );
+const { getShippingZoneExample } = require( '../../data' );
+
+/**
+ * Shipping zone to be created, retrieved, updated, and deleted by the tests.
+ */
+const shippingZone = getShippingZoneExample();
+
+/**
+ * Tests for the WooCommerce Shipping zones API.
+ *
+ * @group api
+ * @group shipping-zones
+ *
+ */
+describe( 'Shipping zones API tests', () => {
+	beforeAll( async () => {
+		// Setup: Delete all pre-existing shipping zones.
+		// Skip deleting shipping zone with id = 0 because it is the "Rest of the world" zone.
```

Comment:
It might be slightly an edge case, but it may be worth adding some tests around the following:

* can't delete the default shipping zone
* the default shipping zone exists and is the only option before we add more

That way we can get coverage if these ever changes, especially since we have some checks in the e2e-utils, for example, expecting this behavior:

https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/e2e-utils/src/flows/with-rest-api.js#L275

Reply:
Thanks @zhongruige! Yep, makes sense. 

I have just added a test to make sure that the default shipping zone remains after deleting all zones. 
I also added a test to make sure that we don't allow it to be modified by the user.

Reply:
Thanks for adding that in!